antora-playbook-local.yml

```diff
@@ -27,7 +27,7 @@ asciidoc:
     idseparator: '-'
     # Variables used in the docs
     page-survey: https://www.surveymonkey.co.uk/r/NYGJNF9
-    hazelcast-cloud: Viridian Cloud
+    hazelcast-cloud: Cloud
```

Comment:
So, do we use just "Cloud" instead of "Hazelcast Cloud" generally in the documentation?

Reply:
The naming discussion is still going on - I agree Hazelcast Cloud would be clearer; however, as the use is varied and there isn't much time for this renaming, I am ignoring it for now and doing minimal changes ;-) 

There will be a follow-up when the naming conventions for all products is confirmed; and at this point consistency will be brought in.

Reply:
Thanks!